### DISCUSSION TITLE
Wert für Stop bei BlindLevelAction einstellbar machen

### DISCUSSION THREAD

**Discussion by kevinhaybach** [Zefau/ioBroker.jarvis#1977] [Ideas / Feature Requests]
Hi,
ich würde gerne für den StopButton eine einstellbare Zahl anstelle von True senden. Dann könnte ich für Up / Down / Stop jeweils die gleiche Variable verwenden. 
Ich habe meine Sichtweise noch im Forum https://forum.iobroker.net/post/874525 etwas ausführlicher beschrieben 

Mit freundlichen Grüßen
Kevin

**Comment by Zefau** [comment]
mittels `stop` in den Geräte-Eigenschaften kannst du dies individualisieren.

**Reply by mcuiobroker** [reply]
Welche Einstellungen bzgl. BlindLevelAction sind denn noch möglich?
Lassen sich "up", "down" und "stop" in Datenpunkt-Eigenschaften anpassen?
`{"stop":3, "up":1, "down":2}`

**Reply by kevinhaybach** [reply]
up und down ist "min" und "max", stop hat bei mir nicht funktioniert

**Reply by Zefau** [reply]
> up und down ist "min" und "max"

korrekt

**Reply by kevinhaybach** [reply]
Nachtrag: nach dem kompletten Reload der Seite funktioniert es mit stop

**Comment by kevinhaybach** [comment]
Mittels stop hat es bei mir nicht funktioniert
<img width="530" alt="image" src="https://user-images.githubusercontent.com/83004345/196505622-cd347c35-a1fd-4bf4-80fc-158c9e8113c2.png">
<img width="785" alt="image" src="https://user-images.githubusercontent.com/83004345/196505676-6ea6786c-ffb0-45cc-8e97-93a3616871cb.png">
<img width="275" alt="image" src="https://user-images.githubusercontent.com/83004345/196505812-ff810dfa-3601-43b8-bf38-927a3a5e7d33.png">
![Stop](https://user-images.githubusercontent.com/83004345/196506157-01fd448f-0a1b-496a-9b07-81c73dc50ef7.gif)

**Reply by Zefau** [reply]
Funktioniert bei mir. Reloade mal die komplette Seite.
